mkosi/__init__.py

```diff
@@ -176,6 +178,18 @@ def remove_packages(state: MkosiState) -> None:
             die(f"Removing packages is not supported for {state.config.distribution}")
 
 
+def check_none_root_populated(state: MkosiState) -> None:
+    """If no distribution was installed, check that root was populated by looking for os-release file."""
+    if state.config.distribution != Distribution.none:
+        return
```

Comment:
This shouldn't be specific to the `none` distribution, we should do this for every distribution.

Reply:
Done